src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -344,8 +351,13 @@ public async Task<RestoreResult> ExecuteAsync(CancellationToken token)
                     }
 
                     // Write the logs into the assets file
-                    var logs = _logger.Errors
-                        .Select(l => AssetsLogMessage.Create(l))
+                    var logsEnumerable = _logger.Errors
```

Comment:
How can we have error messages in addition to the ones provided from the parsing?

Reply:
We can't but since these things happen at different levels of abstraction, we can't guarantee it. The "additional messages" passed in the restore args could only be warnings (if we change something in the future, or someone uses our packages and does it in their own code).